docs/blog/blog.md

```diff
@@ -212,10 +212,10 @@ Encoding nominal categorical variables in unsupervised learning problems while o
 
 #### Steps to get you started:
 
-1. Visit [jupyterhub.datahub.redhat.com](https://jupyterhub.datahub.redhat.com/)
-2. Login with your Kerberos ID
+1. Visit https://jupyterhub-opf-jupyterhub.apps.cnv.massopen.cloud/hub/login
+2. Login with your google account
 3. In this Spawn screen, select s2i-categorical-encoding-notebook:latest, in the memory size select Large ( Memory: 8Gi, CPU 2)
-4. Once your server starts, go into the directory named categorical-encoding-yyyy-mm-dd-hh-mm/demo
+4. Once your server starts, go into the directory named notebooks-yyyy-mm-dd-hh-mm/demo
```

Comment:
This should be reverted. As per https://github.com/operate-first/odh-moc-support/issues/24#issuecomment-735492292 after an image rebuild https://github.com/aicoe-aiops/categorical-encoding/issues/15, we should have the project name included in the folder name. Please check it after the build if it's indeed true.

Reply:
I still see _notebooks_ but it's a matter of time I guess. I'll revert it!

Reply:
yeah, the sesheta is not behaving today https://github.com/aicoe-aiops/categorical-encoding/issues/15 it'll be fixed soon. :slightly_smiling_face: